### DISCUSSION TITLE
Docker Socket Compatibility: Docker-specific tools may not work.

### DISCUSSION THREAD

**Discussion by bathivinod** [podman-desktop/podman-desktop#3180] [General]
While I am starting Podman on Windows, we are getting below error 

`Docker Socket Compatibility: Docker-specific tools may not work.`

![2023-07-12_09-37-37](https://github.com/containers/podman-desktop/assets/27993980/63b36da7-ef1d-4079-83fd-bde7adf4047f)

I have added below environment variable also 
`C:\Users\VINOD\.local\share\containers\podman-desktop\extensions-storage\podman-desktop.compose\bin`

**Comment by mairin** [comment]
@bathivinod we're redesigning the UI for this (check out the Design category in this forum.) It's not really an error... it's just letting you know Podman isn't connected to the Docker socket.

**Reply by bathivinod** [reply]
If `Podman isn't connected to the Docker socket ` means that is also an issue, how to resolve

**Reply by benoitf** [reply]
It means that if you have tools like IDE plugins l, 3rd party tools that connect to barcoded /var/run/docker.sock path it won't work

On windows/macOS there are helpers with 'podman machines command

On Linux you can do a symlink from your podman socket to /var/run/docker.sock but then you could have permissions issues or grant too many permissions.

It depends if you have 3rd party tools

**Reply by bathivinod** [reply]
Ok I understand, Thank you
Where can we find logs for Podman Desktop in Windows

**Comment by afbjorklund** [comment]
The warning looks extra weird, when you get it next to the running Docker...

Like on Ubuntu:

![](https://user-images.githubusercontent.com/10364051/253792720-c6adefb5-10f8-4404-b712-5175cc917d82.png)

The reason that it doesn't work for Podman, is that it is already running Docker. 

But that's different from saying "Docker-specific tools may not work" (for Podman!)
Or that the "Docker socket is not reachable", just because it doesn't understand libpod.

Normally we would prefer to set DOCKER_HOST (or the docker context), to the right one ?

**Comment by mairin** [comment]
@afbjorklund Yehhhh - I'm working on mockups for updating our approach to docker compat in podman this past sprint and this sprint too. Essentially that warning is going to go away and be replaced with something less error message-y, and we'll have a docker compatibility settings page for podman under settings that will hopefully make it less weird. I'm going to post to the discussion board here soon.
